docs/diagrams/UiClassDiagram.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+hide class fields
+hide class methods
+
+package Recipe {
+class RecipeList
+class Recipe
+class Ingredient
+}
+
+package Ui {
+class Ui
+class AddMode
+class Editor
+}
+
+class Storage
+
+Ui ..> CommandResult
+
+AddMode --> Recipe
+AddMode --> Ingredient
+RecipeList *-right-> Recipe
+Recipe *-right-> Ingredient
```

Comment:
```suggestion
Recipe -right-> Ingredient
```
`Recipe` contains attributes of 0..? `ingredient`

Reply:
`Recipe` contains an attribute of `ArrayList<Ingredient>`, and hence I think the multiplicity should be `0..1` instead.